openmic/vggish/__init__.py

```diff
@@ -23,3 +23,41 @@
 
 __pproc__ = Postprocessor(PCA_PARAMS)
 postprocess = __pproc__.postprocess
+
+
+def waveform_to_features(data, sample_rate, compress=True):
+    '''Converts an audio waveform to VGGish features, with or without
+    PCA compression.
+
+    Parameters
+    ----------
+    data : np.array of either one dimension (mono) or two dimensions (stereo)
+
+    sample_rate:
+        Sample rate of the audio data
+
+    compress : bool
+        If True, PCA and quantization are applied to the features.
+        If False, the features are taken directly from the model output
+
+    Returns
+    -------
+    time_points : np.ndarray, len=n
```

Comment:
per #29, we're making a conscious decision to converge to `time_points` then?

Reply:
I literally just copied over from the existing functions.  If we rename, we should do it all at once, and i think it's independent of this pr.

Reply:
👍